internal/proc/proc.go

```diff
@@ -20,6 +20,8 @@ type PetsProc struct {
 	// When the process started
 	StartTime time.Time
 
+	TimeSince time.Duration
```

Comment:
let's make this a struct method instead of a struct property? that way it will always be up to date :)

Reply:
What does that mean?